docs/admin/dokku.md

```diff
@@ -0,0 +1,25 @@
+# Hosting on Dokku
+
+Standards Lab is designed to be hosted on Dokku.
+
+To set up an app, run these commands on the Dokku server:
+
```

Comment:
I think there are missing back ticks around this block? 
\`\`\`
...?

Reply:
No, 4 spaces indent works too, I tested on RTD